doc/api/tty.md

```diff
@@ -211,10 +211,10 @@ added: v0.7.7
 
 * Returns: {number[]}
 
-`writeStream.getWindowSize()` returns the size of the [TTY](tty.md)
+`writeStream.getWindowSize()` returns the size of the [TTY](#)
 corresponding to this `WriteStream`. The array is of the type
 `[numColumns, numRows]` where `numColumns` and `numRows` represent the number
-of columns and rows in the corresponding [TTY](tty.md).
+of columns and rows in the corresponding [TTY](#).
```

Comment:
Non-blocking suggestion: I often find links like this confusing and unhelpful. I never see a link in text and expect that it will be to the start of the document I'm already reading, with the exception of links marked something like "Home" or "Back to top".

```suggestion
of columns and rows in the corresponding TTY.
```